server/metrics/metrics.go

```diff
@@ -0,0 +1,42 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics collects the metrics.
+type Metrics struct {
+	AuthValid,
+	AuthExchange,
+	Errs *prometheus.CounterVec
+}
+
+// M structure to collect all metrics together.
+var M Metrics
+
+// Register metrics and metrics page.
+func Register() {
+	M = Metrics{
+		AuthValid: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "cashier",
+			Subsystem: "auth",
+			Name:      "valid_total",
+			Help:      "Auth Valid calls",
+		}, []string{"module"}),
+		AuthExchange: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "cashier",
+			Subsystem: "auth",
+			Name:      "exchange_total",
+			Help:      "Auth Exchange calls",
+		}, []string{"module"}),
+		Errs: prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Yes. It's a placeholder for discussing how to track errors. Do you want to count all errors, just some or none at all? And to me just an error count per module seems enough; much easier to write alerts on and yet still have them be mildly informative.